crates/cli/src/module_generator.rs

```diff
@@ -36,7 +38,7 @@ use wasm_encoder::{
 //     (export "_start" (func 2))
 //     (data (;0;) "\02\03\0econsole\06log\18function.mjs\0e\00\06\00\a0\01\00\01\00\03\00\00\11\01\a2\01\00\00\008\de\00\00\00B\df\00\00\00\bd*$\01\00\cd(\c0\03\01\00")
 // )
-pub fn generate_module(bytecode: Vec<u8>, js_src: &str) -> Result<Vec<u8>> {
+pub fn generate_module(bytecode: Vec<u8>, js_src: &[u8]) -> Result<Vec<u8>> {
```

Comment:
[Super nit] One thing I didn’t spot earlier: Do we ever need the second memory?

Reply:
I mostly noticed it because neither `wasm2wat` nor `wasmtime` would swallow this without enabling multi memory.